frontend/app/src/store/balances/blockchain-accounts.ts

```diff
@@ -430,17 +432,17 @@ export const useBlockchainAccountsStore = defineStore(
           if (blockchain === Blockchain.ETH) {
             if (modules) {
               const { enableModule } = useSettingsStore();
-              await enableModule({
-                enable: modules,
-                addresses: [address]
-              });
+              startPromise(
```

Comment:
I am not sure about this change, module activation should not be fire and forget, any modules should be activated before doing anything else